toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -180,6 +177,42 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
         isCalloutBeingDisplayed.set(false)
     }
 
+    /**
+     * Creates a Callout at the specified geographical location on the GeoView.
+     *
+     * @since 200.5.0
+     */
+    @Composable
+    private fun CalloutInternal(
+        geoElement: GeoElement,
+        modifier: Modifier = Modifier,
+        tapLocation: Point? = null,
+        content: @Composable BoxScope.() -> Unit
+    ) {
+        var leaderLocation: LeaderLocation? by remember {
+            mutableStateOf(
+                this.computeLeaderLocationForGeoelement(geoElement, tapLocation)
+            )
```

Comment:
Not needed `this` property
```suggestion
            mutableStateOf(computeLeaderLocationForGeoelement(geoElement, tapLocation))
```